sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/TokenCacheImpl.java

```diff
@@ -0,0 +1,153 @@
+package com.azure.identity.implementation;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.microsoft.aad.msal4j.ITokenCache;
+import com.microsoft.aad.msal4j.ITokenCacheAccessAspect;
+import com.microsoft.aad.msal4j.ITokenCacheAccessContext;
+
+import java.nio.charset.StandardCharsets;
+import java.time.OffsetDateTime;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+public class TokenCacheImpl implements ITokenCacheAccessAspect {
```

Comment:
In Memory Cache Impl Class, Remove it if not shipping it.

Reply:
We can remove this.